src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.XboxController;
+import frc.robot.commands.ArcadeDriveCMD;
+import frc.robot.commands.ClimberDownCMD;
+import frc.robot.commands.ClimberStopCMD;
+import frc.robot.commands.ClimberUnlockCMD;
+import frc.robot.commands.ClimberUpCMD;
+import frc.robot.commands.ExampleCommand;
+import frc.robot.controllers.XboxController467.Buttons;
+import frc.robot.controllers.CustomController2020;
+import frc.robot.controllers.XboxController467.Axes;
+import frc.robot.subsystems.Climber;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.ExampleSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+  // The robot's subsystems and commands are defined here...
+  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
+
+  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
```

Comment:
I don't see an ExampleSubsystem or ExampleCommand in the code, this should probably be updated/removed.

Reply:
Did not see [this commit](https://github.com/team467/Robot-Code/commit/87b7a58a51fd87f78da70199d546440d59bfb950), probably should update this pull request with at least that commit when it's stable